.travis.yml

```diff
@@ -16,11 +16,13 @@ before_script:
   - rustup component add rustfmt
   - if [[ ${TRAVIS_RUST_VERSION} == stable ]]; then
       rustup component add clippy;
+      cargo install --force cargo-audit;
```

Comment:
I'm not sure we should enable cargo audit before we create a nightly build that will detect and update vulnerable packages. Do you know a tool that can:

- Run Travis on a nightly basis
- Detect vulnerable packages
- Bump the versions for these packages (this tool should know in which version the vulnerability was fixed) 
- Build and test pickledb to make sure nothing breaks
- Commit the changes to GitHub
- Release a new version

Until we do that, maybe we can just upgrade the versions as you did and release this minor version

Reply:
I couldn't find a crate to run dependency-version-bumping, after having looked for several minutes.
I dropped the changes to the build pipe, pending some automagic tool for dependency bumping